nx-X11/programs/Xserver/xkb/xkb.c

```diff
@@ -6200,7 +6200,7 @@ XkbResetProc(ExtensionEntry *extEntry)
 }
 
 void
-XkbExtensionInit(INITARGS)
```

Comment:
INITARGS remains defined, and used in xkb and other places in the code. Seems inconsistent to remove it only from one place.